packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/lib_auth/services/auth_service.dart

```diff
@@ -0,0 +1,11 @@
+{{> licence.dart }}
+
+import '../repositories/auth_repository.dart';
+
+class AuthService {
```

Comment:
Can you please implement the other methods as well

Reply:
Done